### DISCUSSION TITLE
Frequently Asked Questions — 8bitworkshop documentation

### DISCUSSION THREAD

**Discussion by giscus** [sehugg/8bitworkshop#102] [General]
# Frequently Asked Questions — 8bitworkshop documentation

They’re not actually asked that frequently, but they might be helpful to know. What is this?: It’s a integrated development environment (IDE) for old computers and video games. You can write code f...

https://8bitworkshop.com/docs/docs/faq.html#when-are-you-adding-x-system

**Comment by Spinnetti** [comment]
Just wanted to say hi/thanks.. This tool is really something, packaging many tools in such an easy, user friendly way. My only regret is there's nothing for the Zx81 and its various addons. (I use EightyOne for emulation as it also supports things like various graphic, control and sound addons). Spectrum inclusion is good, but adding the Spectrum Next would make it a one stop shop for me. BTW, haven't looked in details, but does this tool allow for various code and data includes too? Cheers.

**Comment by RepeatGitHub** [comment]
Hi! This is a really cool site, and I love using it! By the way, is there a way to edit existing .nes files? I can't seem to find a way to do so. Again, love the site, and would be really cool to keep working on a project of mine!

**Comment by bkumanchik** [comment]
I love this IDE, I have a quick question, is there a key command to block comment/uncomment code?
